pyproject.toml

```diff
@@ -36,7 +36,7 @@ paths.source = [
 ]
 
 [tool.mypy]
-python_version = "3.7"
```

Comment:
I think the comments should be on the "minimum version" rather than "target version" (regardless of how those settings work).

https://github.com/pylint-dev/pylint-pytest/actions/runs/6637112680/job/18031541929?pr=22